packages/sdk/src/resources-generator.ts

```diff
@@ -86,10 +86,31 @@ export const generateResourcesLoader = ({
   }
 
   let generated = "";
-  generated += `import { loadResource, type System } from "@webstudio-is/sdk";\n`;
+  generated += `import { loadResource, isLocalResource, type System } from "@webstudio-is/sdk";\n`;
+
+  if (hasResources) {
+    generated += `import { sitemap } from "./[sitemap.xml]";\n`;
+  }
+
   generated += `export const loadResources = async (_props: { system: System }) => {\n`;
   generated += generatedVariables;
   if (hasResources) {
+    generated += `
+    const customFetch: typeof fetch = (input, init) => {
```

Comment:
Seems like static function, maybe put it into sdk?

Reply:
yes

Reply:
but probably here will be better as sitedata etc will be arguments